manual/core/logging/README.md

```diff
@@ -0,0 +1,212 @@
+socket...).
+
+Check your logging framework documentation for more information about how to properly configure it.
+You can also find some configuration examples at the end of this page.
+
+Performance tips:
+
+* Use asynchronous appenders; both
+  [Log4J](http://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/AsyncAppender.html) and
+  [Logback](http://logback.qos.ch/manual/appenders.html#AsyncAppender) provide asynchronous
+  appenders that can significantly boost latencies when writing log messages.
+* While the driver does not provide such capability, it is possible for client applications to
+  hot-reload the log configuration without stopping the application. This usually involves JMX and
+  is available for [Logback](http://logback.qos.ch/manual/jmxConfig.html); Log4J provides a
+  `configureAndWatch()` method but it is not recommended to use it inside J2EE containers (see
+  [FAQ](https://logging.apache.org/log4j/1.2/faq.html#a3.6)).
+
+### Taxonomy of driver logs
+
+The driver has a well-defined use for each log level. As an application developer/administrator, you
+should be focusing mostly on the `ERROR`, `WARN` and `INFO` levels.
+
+#### ERROR
+
+Something that renders the driver -- or a part of it -- completely unusable. An action is required
+to fix it: bouncing the client, applying a patch, etc.
+
+#### WARN
+
+Something that the driver can recover from automatically, but indicates a configuration or
+programming error that should be addressed. For example:
+
+```
+WARN  c.d.o.d.i.core.session.PoolManager - [s0] Detected a keyspace change at runtime (<none> =>
+test). This is an anti-pattern that should be avoided in production (see
+'request.warn-if-set-keyspace' in the configuration).
+
+WARN  c.d.o.d.i.c.c.CqlPrepareHandlerBase - Re-preparing already prepared query. This is generally
+an anti-pattern and will likely affect performance. Consider preparing the statement only once.
+Query='...'
+```
+  
+#### INFO
+  
+Something that is part of the normal operation of the driver, but might be useful to know for an
+administrator. For example:
+
+```
+INFO  c.d.o.d.i.c.metadata.MetadataManager - [s0] No contact points provided, defaulting to
+/127.0.0.1:9042
+
+INFO  c.d.o.d.internal.core.time.Clock - Using native clock for microsecond precision
+
+INFO  c.d.o.d.i.c.c.t.DefaultDriverConfigLoader - [s0] Detected a configuration change
+```
+
+#### DEBUG and TRACE
+
+These levels are intended primarily for driver developers; we might ask you to enable them to
+investigate an issue.
+
+Keep in mind that they are quite verbose, in particular TRACE. It's a good idea to only enable them
+on a limited set of categories.
+
+### Logging request latencies
+
+The driver provides a built-in component to log the latency and outcome of every application
+request. See the [request tracker](../request_tracker/#request-logger) page for more details.
```

Comment:
Good idea having a separate document for this, it's a big enough topic to merit its own standalone page 👍